.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+name: Run CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+        DJANGO: ["2.2", "3.2"]
```

```suggestion
        DJANGO: ["3.2"]
```
confirmed with upstream that 2.2 support is not needed